//test exception 8
int b(int n) {
  int x = n / n;
  return b(n - 1);
}

int main() {
  return b(20);
}
